Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -163,3 +163,16 @@ spec:
   cephVersion:
     image: quay.io/ceph/ceph:v17.2.5 # Should match external cluster version
 ```
+
+##  Exporting Rook to another cluster
+
+If you have a multiple K8s cluster running, and want to use `rook-ceph` cluster as the centralize storage place,
+For this we need atleast one cluster to running/deploy as `rook-ceph` cluster,
+And then you can do the following for other clusters:
+
+1) Copy create-external-cluster-resources.py into the directory `/etc/ceph/` of the toolbox.
+   ```console
+   toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+   kubectl -n rook-ceph cp deploy/examples/create-external-cluster-resources.py $toolbox:/etc/ceph
+   ```
+2) Exec to the toolbox pod and execute create-external-cluster-resources.py with needed options to create require [users and keys](/Documentation/CRDs/Cluster/external-cluster.md## Supported Features).
```

Comment:
Since the link is in the same doc, you just need the place to jump without the doc name.
```suggestion
2) Exec to the toolbox pod and execute create-external-cluster-resources.py with needed options to create required [users and keys](#supported-features).
```

Reply:
Done thanks